src/client/views/play.ejs

```diff
@@ -72,7 +72,7 @@
             <p>
                 <%=t('play.guide.rules_paragraphs.0')%>
             </p>
-            <img data-src="/img/game/guide/promotionlines.png" class="img-promotionlines">
+            <img src="/img/game/guide/promotionlines.png" class="img-promotionlines">
```

Comment:
These images are actually intended to have data-src attributes instead of src, this is so that we can dynamically load the guide images only when they open the guide, instead of in page load. This saves bandwidth and improves loading time.

Reply:
If you made this change to fix the images not loading (if they weren't loading), then there's another bug that's causing them to not load when clicking the Guide.